imls-backend/db/migrations/20220811192329_heartbeat.sql

```diff
@@ -0,0 +1,27 @@
+-- migrate:up
+
+CREATE TABLE imlswifi.heartbeats (
+    heartbeat_id SERIAL PRIMARY KEY,
+    fscs_id VARCHAR(16) NOT NULL,
+    sensor_id SERIAL NOT NULL,
+    hourly_ping TIMESTAMP,
```

Comment:
i believe we want `TIMESTAMP WITH TIME ZONE` here. also, do we want this to be automatic? e.g., `DEFAULT CURRENT_TIMESTAMP`? that way, session-counter doesn't have to send the current timestamp (which is a bit redundant anyway)

Reply:
(Also, this should probably be `NOT NULL` as well)